### PR TITLE
doc improvement: add vsc instructions to updating page

### PR DESCRIPTION
**Pull request by Avalei**
Add information about how to update the VSC extension
Direct users to VSC doc site to update west and repos

Ref: NCSDK-12324

Signed-off-by: Deidre Casey <deidre.casey@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/gs_updating.rst
Comment (wiba-nordic) [inline doc/nrf/gs_updating.rst:24]: I think there is just a little bit too much information in one sentence here, and it leaves it ambiguous what to do when the Update button isn't visible.
```suggestion
If you disabled automatic updates, open the :guilabel:`Extensions` tab and locate the |VSC| extension.
The :guilabel:`Update` button becomes visible when an update is available.
Click on the button to install the update.
```
Or something along those lines. 
I'm not quite happy with my own wording.

Comment (Avalei) [reply doc/nrf/gs_updating.rst]: Good point, I have updated this now.

File: doc/nrf/gs_updating.rst
Comment (wiba-nordic) [inline doc/nrf/gs_updating.rst:24]: What do you think, does this now need a paragraph break?
Not because of length, but as a sort of thought reset.

File: doc/nrf/gs_updating.rst
Comment (greg-fer) [inline doc/nrf/gs_updating.rst:17]: Hm, why *in* vsc if the header doesn't have this?

File: doc/nrf/gs_updating.rst
Comment (greg-fer) [inline doc/nrf/gs_updating.rst:20]: This line must match the length of the header.

File: doc/nrf/gs_updating.rst
Comment (greg-fer) [reply doc/nrf/gs_updating.rst:24]: Yes, I agree with @wiba-nordic .

File: doc/nrf/gs_updating.rst
Comment (greg-fer) [inline doc/nrf/gs_updating.rst:29]: Can you link to the related page or at least mention the name of the page?
```suggestion
For detailed instructions, see the page about updating the extension on the `nRF Connect for Visual Studio Code`_ documentation site.
```
(just an example)

Comment (Avalei) [reply doc/nrf/gs_updating.rst:29]: I have been holding off on this since we have been frequently changing the structure of the documentation as it grows. I can add the page in, but this may break often over the next few months as the site grows since we are on a more frequent release schedule than NCS.

File: doc/nrf/gs_updating.rst
Comment (Avalei) [reply doc/nrf/gs_updating.rst]: Fixed

File: doc/nrf/gs_updating.rst
Comment (Avalei) [reply doc/nrf/gs_updating.rst]: Fixed

File: doc/nrf/gs_updating.rst
Comment (Avalei) [reply doc/nrf/gs_updating.rst]: Updated with a paragraph break.

File: doc/nrf/gs_updating.rst
Comment (Avalei) [reply doc/nrf/gs_updating.rst]: Updated with current link. We'll triage later if it breaks.

**Review by wiba-nordic — APPROVED** [review]

**Review by greg-fer — APPROVED** [review]

**Comment by greg-fer** [comment]
@Avalei , remember about the changelog label before requesting merge.

**Pull request merged**
